<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pimak Worlds</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pwclient.qrc">
    <normaloff>:/img/icon.png</normaloff>:/img/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="OgreWidget" name="renderZone" native="true"/>
      <widget class="QTextEdit" name="chatZone">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="message">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="whisperSelector">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="whisper">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actConnect"/>
    <addaction name="actDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actWhisper"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actFirstCam"/>
   <addaction name="actThirdCam"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actConnect">
   <property name="icon">
    <iconset resource="pwclient.qrc">
     <normaloff>:/img/gtk-connect.png</normaloff>:/img/gtk-connect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="actDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="pwclient.qrc">
     <normaloff>:/img/gtk-disconnect.png</normaloff>:/img/gtk-disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="pwclient.qrc">
     <normaloff>:/img/application-exit.png</normaloff>:/img/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actWhisper">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Whisper</string>
   </property>
  </action>
  <action name="actSettings">
   <property name="icon">
    <iconset resource="pwclient.qrc">
     <normaloff>:/img/preferences-desktop.png</normaloff>:/img/preferences-desktop.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="icon">
    <iconset resource="pwclient.qrc">
     <normaloff>:/img/gtk-about.png</normaloff>:/img/gtk-about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About Pimak Worlds...</string>
   </property>
  </action>
  <action name="actFirstCam">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="pwclient.qrc">
     <normaloff>:/img/eyes.png</normaloff>:/img/eyes.png</iconset>
   </property>
   <property name="text">
    <string>First person camera</string>
   </property>
  </action>
  <action name="actThirdCam">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="pwclient.qrc">
     <normaloff>:/img/camera-video.png</normaloff>:/img/camera-video.png</iconset>
   </property>
   <property name="text">
    <string>Third person camera</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OgreWidget</class>
   <extends>QWidget</extends>
   <header>OgreWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="pwclient.qrc"/>
 </resources>
 <connections/>
</ui>
